charts/vector/values.yaml

```diff
@@ -1,75 +1,75 @@
-## Default values for Vector
-## See Vector helm documentation to learn more:
-## https://vector.dev/docs/setup/installation/package-managers/helm/
+# Default values for Vector
+# See Vector helm documentation to learn more:
+# https://vector.dev/docs/setup/installation/package-managers/helm/
 
-# nameOverride -- Override name of app
+# nameOverride -- Override the name of resources.
 nameOverride: ""
 
-# fullnameOverride -- Override the full qualified app name
+# fullnameOverride -- Override the full name of resources.
 fullnameOverride: ""
 
-# role -- Role for this Vector (possible values: Agent, Aggregator, Stateless-Aggregator)
-## Ref: https://vector.dev/docs/setup/deployment/roles/
-## Each role is created with the following workloads:
-## Agent - DaemonSet
-## Aggregator - StatefulSet
-## Stateless-Aggregator - Deployment
+# role -- [Role](https://vector.dev/docs/setup/deployment/roles/) for this Vector instance, valid options are:
+# "Agent", "Aggregator", and "Stateless-Aggregator".
+
+# Each role is created with the following workloads:
+# Agent = DaemonSet
+# Aggregator = StatefulSet
+# Stateless-Aggregator = Deployment
 role: "Aggregator"
 
-# rollWorkload -- Add a checksum of the generated ConfigMap to workload annotations
+# rollWorkload -- Add a checksum of the generated ConfigMap to workload annotations.
 rollWorkload: true
 
-# commonLabels -- Add additional labels to all created resources
+# commonLabels -- Add additional labels to all created resources.
 commonLabels: {}
 
-## Define the Vector image to use
+# Define the Vector image to use.
 image:
-  # image.repository -- Override default registry + name for Vector
+  # image.repository -- Override default registry and name for Vector's image.
   repository: timberio/vector
-  # image.pullPolicy -- Vector image pullPolicy
+  # image.pullPolicy -- The [pullPolicy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for
+  # Vector's image.
   pullPolicy: IfNotPresent
-  # image.pullSecrets -- Agent repository pullSecret (ex: specify docker registry credentials)
-  ## Ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # image.pullSecrets -- The [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+  # to reference for the Vector Pods.
   pullSecrets: []
-  # image.tag -- Vector image tag to use
-  # @default -- Chart's appVersion
+  # image.tag -- The tag to use for Vector's image.
+  # @default -- Derived from the Chart's appVersion.
```

Comment:
Just noticing "Chart" is capitalized. Is that like a standard helm thing to do?

Reply:
Unsure honestly... it's sorta a proper noun?

> Helm helps you manage Kubernetes applications — Helm Charts help you define, install, and upgrade even the most complex Kubernetes application.

from: https://helm.sh/